index.html

```diff
@@ -1,3 +1,71 @@
-﻿
+﻿<!doctype html>
+<html lang="en">
+  <head>
+    <title>The Best Chocolate Chip Cookies</title>
+  </head>
 
+  <body>
+    <header>
+      <h1>The Best Chocolate Chip Cookies</h1>
+    </header>
 
+    <section>
+      <h4>Recipe by Grandma</h4>
+
+      <img src="./img/cookies.jpg" alt="Mama's Cookies">
+      <p>
+        <h5>Prep Time: 45 min</h5>
+      </p>
+
+    </section>
+
+    <section>
+      <h3>How to make Mama’s Recipe:</h3>
+        <p>
```

Comment:
Instead of using a bunch of `p` tags and typing out the numbers, you can use an ordered list! It automatically creates the numbers for you. Something like:
```
<ol> <!-- ordered list tab -->
  <li>Preheat oven to 350 degrees</li> <!-- a list item -->
  <li>Beat butter and sugars together until light and fluffy</li>
</ol>
```